test/JDBC/input/constraint_column_usage-vu-verify.sql

```diff
@@ -7,7 +7,7 @@ go
 Use master;
 go
 
-SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME NOT LIKE 'sys%' ORDER BY COLUMN_NAME;
+SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME LIKE 'constraint_column_usage_vu_prepare%' ORDER BY COLUMN_NAME;
```

Comment:
what is the purpose of this test now? test is creating all tables in db_constraint_column_usage_vu_prepare DB, so will never have tables with db_constraint_column_usage_vu_prepare% name in master. I thin it should be removed

Reply:
This test is/was ensuring that view shows only current db's object info and doesn't leak other db's info.